src/core/loading/mod.rs

```diff
@@ -1,14 +1,13 @@
+use crate::AppState;
```

Comment:
Crate-local imports should come after 3rd-party imports, with a blank line separating, like before.

This comment applies to the other places this was changed as well.

Reply:
I'll see if I can just add a `.rustfmt` for that

Reply:
Update: yes, `rustfmt.toml` has an option for that, but it's nightly only. I've applied it but have not added it to the repo because it will generate warnings on stable.

Reply:
My `rustfmt.toml`:
```toml
# Stable
max_width = 100
use_field_init_shorthand = true
use_try_shorthand = true

# Unstable
group_imports = "StdExternalCrate"
imports_granularity = "Item"
imports_layout = "HorizontalVertical"
match_block_trailing_comma = true
normalize_doc_attributes = true
```
This is my personal preference, not a suggestion for this repo, necessarily.